pkg/console/subresource/consoleserver/types.go

```diff
@@ -202,6 +202,16 @@ const (
 	PerspectiveAccessReview PerspectiveState = "AccessReview"
 )
 
+// PerspectiveID defines the id of the perspective.
+// "admin" is the id of the admin perspective.
+// "dev" is the id of the developer perspective.
+type PerspectiveID string
+
+const (
+	PerspectiveIDAdmin PerspectiveID = "admin"
+	PerspectiveIDDev   PerspectiveID = "dev"
+)
```

Comment:
Shall we limit the valid values for `.spec.customization.perspectives`? 
```
$ oc patch console.operator cluster -p '{"spec":{"customization":{"perspectives":[{"id":"dev","visibility":{"state":"Disabled"}},{"id":"admin","visibility":{"state":"Disabled"}},{"id":"test","visibility":{"state":"Disabled"}}]}}}' --type merge
console.operator.openshift.io/cluster patched
[yapei@yapei-mac frontend (support)]$ oc get console.operator cluster -o json | jq .spec.customization.perspectives
[
  {
    "id": "dev",
    "visibility": {
      "state": "Disabled"
    }
  },
  {
    "id": "admin",
    "visibility": {
      "state": "Disabled"
    }
  },
  {
    "id": "test",
    "visibility": {
      "state": "Disabled"
    }
  }
]
```

Reply:
@yapei Unfortunately we can't since we don't know the IDs of perspectives that can be contributed by plugins

Reply:
thank you @spadgett